src/ReqResAutomation/CreateIdTest.java

```diff
@@ -0,0 +1,45 @@
+  package ReqResAutomation;
+
+  import io.restassured.builder.RequestSpecBuilder;
+  import io.restassured.path.json.JsonPath;
+  import io.restassured.specification.RequestSpecification;
+  import static io.restassured.RestAssured.given;
+  import Resources.BaseUrl;
+  import SerializationPojo.CreatePojo;
+
+  public class CreateIdTest
+  {
+    public static void main(String[] args)
+    {	
+		
+	 RequestSpecification req_spec = new RequestSpecBuilder().setBaseUri(BaseUrl.baseUri()).build();
```

Comment:
Indentation is off. Figure out how to make your IDE do auto-indent. That will be a lot more convenient than manually indenting correctly.